src/pages/networks/forms/NetworkSelector.tsx

```diff
@@ -4,31 +4,47 @@ import Loader from "components/Loader";
 import { useQuery } from "@tanstack/react-query";
 import { queryKeys } from "util/queryKeys";
 import { fetchNetworks } from "api/networks";
+import { fetchProject } from "api/projects";
 
 interface Props {
   isDisabled: boolean;
   project: string;
   props?: Record<string, unknown>;
 }
 
-const NetworkSelector: FC<Props> = ({ isDisabled, project, props }) => {
-  const { data: networks = [], isLoading } = useQuery({
-    queryKey: [queryKeys.networks],
-    queryFn: () => fetchNetworks(project),
+const NetworkSelector: FC<Props> = ({
+  isDisabled,
+  project: projectName,
+  props,
+}) => {
+  const { data: networks = [], isLoading: isNetworkLoading } = useQuery({
+    queryKey: [queryKeys.networks, "default"],
+    queryFn: () => fetchNetworks("default"),
   });
 
-  const options = networks.map((network) => {
+  const { data: project, isLoading: isProjectLoading } = useQuery({
+    queryKey: [queryKeys.projects, projectName],
+    queryFn: () => fetchProject(projectName),
+  });
+
+  const availableNetworks =
+    project?.config?.["restricted.networks.uplinks"]?.split(",") ||
+    networks
+      .filter((network) => ["bridge", "physical"].includes(network.type))
```

Comment:
I don't know if this is expected to be stable, but just in case I would suggest declaring a `const` for it above:
```
const UI_NETWORK_TYPES = ["bridge", "physical"];
```
And then this line becomes:
```suggestion
      .filter((network) => UI_NETWORK_TYPES.includes(network.type))
```
I think it's cleaner this way.

Reply:
This component is heavily opinionated to be an OVN uplink selector. The types here are the ones supported by LXD as OVN uplinks. I don't think they will change much. I'll rename the component.

Reply:
Regardless of that, it's still tidier to have them declared in a `const` rather than hardcoded in the middle of the logic.